src/NetworkTest/testConnectivity/index.ts

```diff
@@ -181,6 +181,9 @@ function checkCreateLocalPublisher(
         if (!Object.keys(availableDevices.video).length) {
           publisherOptions.videoSource = null;
         }
+        if (options && options.scalableVideo) {
```

Comment:
Nit: more idiomatic.

```
if (options?.scalableVideo) {
  <snip>
}
```

Reply:
We'd have to update our linter version to support this. And then we would want to make similar changes to use this syntax.

Reply:
I think this will introduce a bug.  If `options.scalableVideo` is `false` then it'll never be set as a publisher option.

Perhaps this will work:
```
if (options && typeof options.scalableVideo === 'boolean') {
  <snip>
}
```

Reply:
Ah, good catch.